01-starting-with-data.Rmd

```diff
@@ -41,7 +41,7 @@ To do all that, we'll have to learn a little bit about programming.
 To load our inflammation data, first we need to tell our computer where is the file that contains the values. We have been told its name is `inflammation-01.csv`. This is very important in R, if we forget this step we’ll get an error message when trying to read the file. We can change the current working directory using the function `setwd`. For this example, we change the path to the directory we just created:
 
 ```{r,eval=FALSE}
-setwd("~/Desktop/r-novice-inflammation/)
+setwd("~/Desktop/r-novice-inflammation/")
```

Comment:
This has been fixed by #184. It's also probably the source of the merge conflict, so I suggest removing it from this PR.